apps/admin/backend/src/util/err_cdf.ts

```diff
@@ -0,0 +1,277 @@
+      {
+        '@type': 'ElectionResults.BallotMeasureSelection',
+        '@id': contest.yesOption.id,
+        Selection: asInternationalizedText(contest.yesOption.label),
+        VoteCounts: [
+          {
+            '@type': 'ElectionResults.VoteCounts',
+            Count: results.yesTally,
+            GpUnitId: contest.districtId,
+            Type: ResultsReporting.CountItemType.Total,
+          },
+        ],
+      },
+      {
+        '@type': 'ElectionResults.BallotMeasureSelection',
+        '@id': contest.noOption.id,
+        Selection: asInternationalizedText(contest.noOption.label),
+        VoteCounts: [
+          {
+            '@type': 'ElectionResults.VoteCounts',
+            Count: results.noTally,
+            GpUnitId: contest.districtId,
+            Type: ResultsReporting.CountItemType.Total,
+          },
+        ],
+      },
+    ],
+    OtherCounts: [
+      {
+        '@type': 'ElectionResults.OtherCounts',
+        GpUnitId: contest.districtId,
+        Overvotes: results.overvotes,
+        Undervotes: results.undervotes,
+      },
+    ],
+  };
+}
+
+function getCandidateContest(
+  contest: CandidateContest,
+  results: Tabulation.CandidateContestResults
+): ResultsReporting.CandidateContest {
+  return {
+    '@type': 'ElectionResults.CandidateContest',
+    '@id': contest.id,
+    Name: contest.title,
+    ElectionDistrictId: contest.districtId,
+    VotesAllowed: results.votesAllowed,
+    ContestSelection: Object.values(results.tallies).map((candidateTally) => ({
+      '@type': 'ElectionResults.CandidateSelection',
+      '@id': candidateTally.id,
+      IsWriteIn: candidateTally.isWriteIn,
+      VoteCounts: [
+        {
+          '@type': 'ElectionResults.VoteCounts',
+          GpUnitId: contest.districtId,
+          Count: candidateTally.tally,
+          Type: ResultsReporting.CountItemType.Total,
+        },
+      ],
+    })),
+    OtherCounts: [
+      {
+        '@type': 'ElectionResults.OtherCounts',
+        GpUnitId: contest.districtId,
+        Overvotes: results.overvotes,
+        Undervotes: results.undervotes,
+      },
+    ],
+  };
+}
+
+type ReportContest =
+  | ResultsReporting.BallotMeasureContest
+  | ResultsReporting.CandidateContest;
+
+function getContests(
+  election: Election,
+  electionResults: Tabulation.ElectionResults
+): ReportContest[] {
+  const reportContests: ReportContest[] = [];
+
+  for (const contest of election.contests) {
+    const contestResults = electionResults.contestResults[contest.id];
+    assert(contestResults);
+    if (contest.type === 'yesno') {
+      assert(contestResults.contestType === 'yesno');
+      reportContests.push(getBallotMeasureContest(contest, contestResults));
+    } else {
+      assert(contestResults.contestType === 'candidate');
+      reportContests.push(getCandidateContest(contest, contestResults));
+    }
+  }
+
+  return reportContests;
+}
+
+/**
+ *
+ */
+export function getElectionResultsReport({
```

Comment:
Nit: consider more descriptive words than `get` to start functions (e.g. I think `build` makes more sense here)